tests/nosetests/pyanaconda_tests/module_payload_shared.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2019  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Jiri Konecny <jkonecny@redhat.com>
+#
+import unittest
+from mock import Mock
+
+from tests.nosetests.pyanaconda_tests import check_kickstart_interface
+from pyanaconda.modules.payload.payload_interface import PayloadInterface
+from pyanaconda.modules.payload.payload import PayloadModule
+
+
+class PayloadHandlerBase(unittest.TestCase):
```

Comment:
Having all the kickstart tests for all the handlers in one file would be overwhelming. I want to avoid that if possible and I also want to avoid copy-paste code.

Yeah it will be evaluated but when there are no tests I don't see a problem. The other possible solution is to have this as a mixin class. I don't have problem with that but the separate file for this is still required. This file can be moved to another directory if you want?

Reply:
Ok, create at least a mixin class, please.

Reply:
Fixed.